SteamReview/SteamReview/Steam.cpp

```diff
@@ -148,21 +167,20 @@ void FSteam::AppRun()
 }
 
 //---------------------------------------------------------------
-int FSteam::ValidateInput(int InputValue, const int LowBoundValue, const int HighBoundValue)
+void FSteam::ValidateInput(int& InputValue, const int LowBoundValue, const int HighBoundValue)
 {
-
+	std::cin >> InputValue;
 	while (std::cin.fail() || (InputValue < LowBoundValue) || (InputValue > HighBoundValue))
 	{
 		std::cin.clear();
 		std::cin.ignore(100, '\n');
 		std::cout << "Invalid Option, please enter a valid Option " << std::endl;
 		std::cin >> InputValue;
 	}
-	return InputValue;
 };
 
 //--------------------------------------------------------------
-FGame FSteam::AddNewGame()
+void FSteam::AddNewGame(FGame& InGame)
```

Comment:
Doesn't seem like you'd want to change this game at all, would you?

In that case, this would be a good place to use a _const_ reference